rpc_jobs/params.yml

```diff
@@ -5,6 +5,14 @@
       - string:
           name: REGION
```

Comment:
I think it would be worth changing this to `REGIONS`, the current names hides that this is a list.

Reply:
Changed to REGIONS, required changes to other jobs.